packages/addressBadge/AddressBadge.tsx

```diff
@@ -14,6 +19,7 @@ export const AddressBadge = forwardRef(
       <AddressBadgeStyle
         symbols={isMobile ? symbolsMobile : symbolsDesktop}
         address={address ?? ''}
+        {...rest}
```

Comment:
```suggestion
        {...rest}
        symbols={isMobile ? symbolsMobile : symbolsDesktop}
        address={address ?? ''}
```

Reply:
Discussed in DM, will keep {...rest} at the end